dm/load_worker_test.go

```diff
@@ -0,0 +1,52 @@
+package dm
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/pingcap/tiflow/dm/pkg/log"
+	"github.com/pingcap/tiflow/dm/pkg/utils"
+	"github.com/stretchr/testify/require"
+
+	"github.com/hanfei1991/microcosm/lib"
+	"github.com/hanfei1991/microcosm/lib/registry"
+	dcontext "github.com/hanfei1991/microcosm/pkg/context"
+)
+
+func TestLoadWorker(t *testing.T) {
+	// only support manually prepare environment and test
```

Comment:
Does `manually prepare environment` mean after we run something else, then we can run the following test case.

Reply:
currently we need start a MySQL at port 3306 and TiDB at port 4000, then run the TestDumpWorker and TestLoadWorker in sequence

Reply:
Better to write the above test procedure to the comment of this case.